DevEdu.API/DevEdu.Business/Services/StudentAnswerOnTaskService.cs

```diff
@@ -77,6 +79,13 @@ public void UpdateStudentAnswerOnTask(int taskId, int studentId, StudentAnswerOn
 
         public void AddCommentOnStudentAnswer(int taskstudentId, int commentId) => _studentAnswerOnTaskRepository.AddCommentOnStudentAnswer(taskstudentId, commentId);
 
+        public List<StudentAnswerOnTaskDto> GetAnswersForStudentInGroup(int userId)
+        {
+            int groupId = _groupRepository.GetPresentGroupForStudentByUserId(userId);
+            var listTasksForGroup = _groupRepository.GetTaskGroupByGroupId(groupId);
+            var dto = _studentAnswerOnTaskRepository.GetStudentAnswersOnTasksByGroupId(groupId, userId, listTasksForGroup);
 
+            return dto;
```

Comment:
you can change this method after changes for that:
![image](https://user-images.githubusercontent.com/18329725/127050133-ff6920a3-e7ea-4a75-b991-3007fee36f3e.png)

please create a stored procedure with userId as an input parameter
and list of Task_Student entities as a result